lamindb/curators/core.py

```diff
@@ -464,7 +502,11 @@ def __init__(
                     pandera_columns[feature.name] = pandera.Column(
                         dtype=None,
                         checks=pandera.Check(
-                            check_dtype(feature.dtype),
+                            check_dtype(
+                                "str"
```

Comment:
Why is this "str" if dtype startswith "cat"?

Reply:
It makes a big difference and we have to be consistent about validating categoricals vs strings of course.